docs/CALIBRATION_GUIDE.md

```diff
@@ -0,0 +1,88 @@
+
+<!-- TODO: Insert screenshot of an undersized experimental setup -->
+
+Last but not least, it is vital to have the cameras synchronized when recording. This can be achieved with a hardware controller (such as a Rasberry Pi) and a pulse emitter. The synchronized videos must then be saved by an appropriate acquisition software and compressed if necessary. For the benchmark dataset, the camera setup consisted of 5 (4 for some sessions) FLIR Blackfly S Mono cameras, each with ThorLabs fixed focal length 3.5 or 4.5 mm lenses (MVL4WA/ 5WA). Cameras were triggered with a strobe pulse from a tdt rx8. Data was acquired with spinview software and then compressed with ffmpeg. 
+
+## Calibration Workflow 
+
+Calibration should be carried out each time before the start of a recording session. If one is recording multiple sessions within the same day, it is fine to calibrate only at the beginning of the first session, given that the camera setup is not altered between sessions. If the camera setup is altered between sessions, calibration must be carried out once again. The workflow is usually conducted as follows.
+
+1. Check camera synchronization and setup stability. The cameras should not move throughout the rest of the workflow and during the subsequent experiments. 
+
+2. Place the calibration board in front of the camera setup such that it is clearly visible to as many cameras as possible. 
+
+3. Start recording. 
+
+4. Move the calibration board around the setup to expose it to other cameras from which it may have been occluded. Even if the board is visible to all cameras one should still move the board around so that marker detection can be robust to variability in board position. Also, it is critical to move the board around at a pace that is suitable for the framerate of the cameras, as blurry frames lead to poor detection. 
+
+5. Stop recording. 
+
+6. Check the videos for proper synchronization and visibility, troubleshoot if necessary. 
+
+Alternatively, instead of taking videos for board calibration, one could take synchronized images from each view and then use sleap-anipose to stitch the frames into a movie. Refer to the [function documentation](sleap_anipose/calibration.py) for more details. 
+
+7. Organize the different videos into their proper view subfolders as explained in [FOLDER_STRUCTURE.md](sleap-anipose/docs/FOLDER_STRUCTURE.md).
+
+8. Run the calibration function from either the command line or a script.
+
+```
+slap-calibrate -- session my/path --board my/path/board.toml --calib_fname my/path/calibration.toml --metadata_fname my/path/calibration.metadata.h5 --histogram_path my/path --reproj_path my/path
+```
+
+```python
+
+import sleap-anipose as slap 
+cgroup, metadata = slap.calibration.calibrate("my/path", "my/path/board.toml", "my/path/calibration.toml", "my/path/calibration.metadata.h5", "my/path", "my/path")
```

Comment:
Fix import and call:
```
import sleap_anipose as slap 
cgroup, metadata = slap.calibrate("my/path", "my/path/board.toml", "my/path/calibration.toml", "my/path/calibration.metadata.h5", "my/path", "my/path")
```

If `calibrate` isn't at the top-level, import it in `__init__.py`.

Reply:
resolved